pydantic/config.py

```diff
@@ -6,12 +6,11 @@
 from typing_extensions import Literal, TypeAlias, TypedDict
 
 from ._migration import getattr_migration
-from .deprecated.config import BaseConfig, Extra
 
 if TYPE_CHECKING:
     from ._internal._generate_schema import GenerateSchema as _GenerateSchema
 
-__all__ = 'BaseConfig', 'ConfigDict', 'Extra'
+__all__ = ('ConfigDict',)
```

Comment:
Is this change technically breaking, does it need a module-level `__getattr__`?

Reply:
it's covered by the migrations `__getattr__` I believe.

Reply:
it is.